utils/R/remote.R

```diff
@@ -201,7 +201,14 @@ remote.execute.R <- function(script, host="localhost", user=NA, verbose=FALSE, R
   } else {
     remote <- ifelse(is.na(user), host, paste(user, host, sep='@'))[[1]]
     result = system2('ssh', c('-T', remote, R, "--vanilla"), stdout=verbose, stderr=verbose, input=input)
-    remote.copy.file(host, tmpfile, user, "localhost", tmpfile)
+    
+    # Tunnel is hardcoded in for geo ... where should this be specified?
+    host_list <- list(name=host, user=user, tunnel="/tmp/geo.tunnel")
+    remote.copy.from(host_list, tmpfile, tmpfile, delete=TRUE)
+    
```

Comment:
@robkooper could you check what @bcow is doing here, I'm unfamiliar with how you have tunnels set up.